lib/addons/amp/index.ts

```diff
@@ -93,10 +132,37 @@ export class AmpAddOn implements ClusterAddOn {
             manifest,
             values
         };
-
         const kubectlProvider = new KubectlProvider(clusterInfo);
         const statement = kubectlProvider.addManifest(manifestDeployment);
 
+        const ampRules = this.ampAddOnProps.ampRules;
+        if (ampRules !== undefined){
```

Comment:
What will happen if `openTelemetryCollectorManifestPath` and `ampRules` are passed together? Will this work. Have you tried to test those boundary conditons?

Reply:
yes that works, done.